packages/grafana-ui/src/components/CustomScrollbar/CustomScrollbar.tsx

```diff
@@ -64,22 +62,6 @@ export const CustomScrollbar = ({
     }
   }, [scrollTop]);
 
-  /**
-   * Special logic for doing a update a few milliseconds after mount to check for
-   * updated height due to dynamic content
-   */
-  useEffect(() => {
```

Comment:
Did you want to remove this?

Reply:
thanks for spotting this - all fixed now 🙏